src/mlpack/bindings/go/print_doc_functions_impl.hpp

```diff
@@ -77,6 +77,26 @@ inline std::string PrintValue(const T& value, bool quotes)
   return oss.str();
 }
 
+/**
+ * Given a vector parameter type, print the corresponding value.
+ */
+template<typename T>
+inline std::string PrintValue(const std::vector<T>& value, bool quotes)
```

Comment:
I'm wondering does any binding reuse some functionality from another binding implementation, looks like the `PrintValue` implementation is the same for each binding, so maybe we could reuse it?

Reply:
Yeah, I think this may be true throughout.  Tomorrow I will check more thoroughly on this and open an issue if so (it's a nice starter issue, simple refactoring).  In this PR I don't think it's necessary though.

Reply:
Sounds fine for me.

Reply:
Actually, it's a good thing I looked through again, because I realized that for many languages, I was not delimiting the arrays right.  I also looked through the other `PrintValue()` overload and found a lot of differences... Julia uses backticks for quotes; for Go we use quotes; for the CLI and Python we use single-quotes.  So I think there is not much refactoring opportunity.

@jeffin143 I added some suggestions to fix all my little errors. :)